### DISCUSSION TITLE
Single click to open file in treemacs?

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#45] [Q&A] [answered]
Hi.

Is there any way to make Treemacs reliably open a selected file with one single left click?
The `open` in the context menu (right-click) works fine. but for the left click, I need to press it more than 3-4 times to open a file. 
Sometimes it works, sometimes it doesn't. As shown in the gif below.

![ezgif com-gif-maker](https://user-images.githubusercontent.com/17734314/230551776-09085cae-b8d2-418e-8f61-5409eac4b23d.gif)

**Comment by abougouffa** [comment, marked as answer]
I have no idea!

I don't use `treemacs` regularly, and when I do, I use it only from the keyboard, so I didn't need to play with mouse settings.

It seems to be a `treemacs` issue, [I did verify the project's README](https://github.com/Alexander-Miller/treemacs#mouse-interface), it seems to be a bug as it is not working as expected.
